wp-content/themes/pub/wporg-learn-2020/template-parts/component-post-card-footer-wporg_workshop.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use function WPOrg_Learn\Post_Meta\get_workshop_duration;
+
+$topics = wp_get_post_terms( $post->ID, 'topic', array( 'fields' => 'names' ) );
+$topics_string = implode( ', ', array_map( 'esc_html', $topics ) );
+$languages = esc_html( $post->video_language );
+$duration = get_workshop_duration( $post, 'string' );
+
+$post_tags = array(
+	'testimonial' => $topics_string,
+	'clock' => $duration,
+	'admin-site-alt3' => $languages,
+)
```

Comment:
Just a suggestion— it would be nice to add an icon onto any captioned workshops (like the language section, it could say which languages are available). I'd say the `tesimonial` icon makes the most sense for captions, but it's already used for topics. Thoughts @melchoyce?

![Screen Shot 2020-08-31 at 3 12 37 PM](https://user-images.githubusercontent.com/541093/91757504-6a5b1100-eb9c-11ea-9f0a-82013633d5db.png)

Reply:
Can we use the category icon for topics instead of the testimonial icon?

Reply:
Yeah, the category icons does work better! Made the update.